### ISSUE TITLE
experience.yml.new

### ISSUE THREAD

**Issue by andrewkm** [mcMMO-Dev/mcMMO-Classic#14]
the experience.yml that is located here:
https://github.com/mcMMO-Dev/mcMMO-Classic/blob/master/src/main/resources/experience.yml

Is different than the experience.yml.new that the plugin generates which is:
https://pastebin.com/kbP5XFM0

Due to this the plugin will continue to endlessly generate the experience.yml.new file regardless if you're using the latest one according to the github /src/main/resources/experience.yml

Further more, there are some things broken in experience.yml
The github one has Nether_Wart_Ripe meanwhile the experience.yml.new file generated by the plugin has both Nether_Wart_Ripe and Nether_Warts_Ripe. Which by the way both aren't even valid items ...

TL;DR experience.yml needs a cleaning.

**Comment by t00thpick1** [comment]
Thanks for the report

**Comment by andrewkm** [comment]
waiting on config update.

**Comment by t00thpick1** [comment]
I had looked into this, sorry I didn't respond.  The current experience.yml is correct.  It is obsolete kesy from prior to the auto-update that are the issue here.  (The _Ripe suffix is part of the config system that allows you to specify rewards only for fully grown crops rather than any stage).  The auto-update system does not handle obsolete keys well.
